aep/general/0231/aep.md.j2

```diff
@@ -126,24 +115,66 @@ message BatchGetBooksResponse {
 }
 ```
 
-- The response message **must** include one repeated field corresponding to the
-  resources being retrieved.
+- The response message **must** include one repeated field where its items may
+  either be a retrieved resource or an error structure describing an error that
+  occurred attempting to retrieve the resource. The error alternative would
+  only be present for non-transactional batch gets.
 
 {% tab oas %}
 
-- The response schema **must** be an array with each item containing one of the
-  requested resources.
+- The response schema **must** be `type: object` with a single array property
+  with each item containing one of the requested resources.
+
+```json
+{
+  "results": [
+    {
+      "name": "publishers/lacroix/books/les-mis",
+      "isbn": "978-037-540317-0",
+      "title": "Les Misérables",
+      "authors": ["Victor Hugo"],
+      "rating": 9.6
+    },
+    {
+      "code": "NotFound",
+      "message": "The resource is not available"
+    }
+    {
+      "name": "publishers/lacroix/books/hunchback-of-notre-dame",
+      "isbn": "978-140-274575-1",
+      "title": "The Hunchback of Notre Dame",
+      "authors": ["Victor Hugo"],
+      "rating": 9.3
+    }
+  ]
+}
+```
+
+- The array of resources **must** be named `results` and contain resources with
+  no additional wrapping.
+- There must not be a `nextPageToken` field as batch get operations are not
+  pageable.
 
 {% endtabs %}
 
-- The order of resources in the response **must** be the same as the paths in
-  the request.
+- The order of resources/error objects in the response **must** be the same as
+  the paths in the request.
+
+### Support for transactional get
+
+The batch get method may support a transactional form of operation where the
+get either succeeds for all requested resources or fails. When supported, the
+method should define a boolean parameter `asTransaction` that the user must
```

Comment:
I think `transactional` might be more consistent with the general AEP style than `asTransaction`, since the AEPs try to avoid prepositions in field names in general.  (But I don't remember if we already discussed this...)